cli/module_graph2.rs

```diff
@@ -692,6 +692,12 @@ impl Graph2 {
     self.modules.contains_key(s)
   }
 
+  /// Consume graph and return list of all module specifiers
+  /// contained in the graph.
+  pub fn get_modules(&self) -> Vec<ModuleSpecifier> {
```

Comment:
Can we put this in alphabetical order? I realise `contains_module` should come after `check` but that was an oversight/mistake on my part.